linked-list.rb

```diff
@@ -63,14 +63,45 @@ def display
    end
 
    def include?(key)
+     current = @head
+    #while we have nodes to traverse
+     while current != nil
+     #when we find the key
+       return true if current.value == key
+       #reassign currrent
+       current = current.next_node
+     end
+     puts "The list does not include #{key}"
+     return false
    end
```

Comment:
yes! this looks nice! the only thing is that you shouldn't "puts" anything because I did not ask for that, which means that you are creating unintended side-affects for the method that may not be expected.